frontend-vanilla.js

```diff
@@ -11,6 +11,22 @@ module.exports = {
     "plugin:prettier/recommended",
   ],
   ignorePatterns: ["dist", ".eslintrc.cjs"],
-  plugins: [],
-  rules: {},
+  plugins: ["check-file"],
+  rules: {
+    "check-file/filename-naming-convention": [
+      "error",
+      {
+        "**/*.{js,ts,tsx}": "KEBAB_CASE"
+      },
+      {
+        "ignoreMiddleExtensions": true
+      }
+    ],
```

Comment:
Har inte kollat så noga men säger den här att filer måste vara `kebab-case`? Eller att kebab case är error?

Kanske bra med lite tester?

Reply:
Den säger att filerna måste heta kebab-case, annars blir det error. Jag ska se om jag kan få in några tester